server/index.js

```diff
@@ -60,6 +61,23 @@ app.get('/api/screenshots/:id', (req, res, next) => {
   ;
 });
 
+app.post('/create-payment-intent', async (req, res) => {
+  // const { items } = req.body;
+
+  // Create a PaymentIntent with the order amount and currency
+  const paymentIntent = await stripe.paymentIntents.create({
+    amount: 2000,
```

Comment:
This should be the user's real total. You can use the request token to query the database and get their items.